backend/account/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from .views import custom_logout
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+)
+
+urlpatterns = [
+    path('logout/', custom_logout, name='logout'),
```

Comment:
might be better to name to something like logout_veiew if its func or LogoutView got class